src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book, onDelete }) => {
+  const handleDelete = () => {
+    onDelete(book.id);
+  };
+
+  return (
+    <div>
+      <h2>{book.title}</h2>
```

Comment:
- [ ] the title of the book is not showing at the moment because the prop that you are passing doesn't have a title in as a key in the object, the key you want to use in this case is name because that's what it's named in your books.js file